server.h

```diff
@@ -58,19 +63,31 @@ class CTFOServer final {
  public:
   using Storage = CTFOStorage<SherlockStreamPersister>;
   using MidichloriansServer = current::midichlorians::server::MidichloriansHTTPServer<CTFOServer>;
+  using RESTStorage = RESTfulStorage<Storage,
+#ifdef USE_BASIC_HYPERMEDIA
+                                     current::storage::rest::Hypermedia>;
+#else
+                                     current::storage::rest::AdvancedHypermedia>;
+#endif
 
   explicit CTFOServer(const std::string& cards_file,
                       int port,
                       const std::string& storage_file,
                       int midichlorians_port,
                       const std::string& midichlorians_file,
+                      int rest_port,
+                      const std::string& rest_url_prefix,
                       const std::chrono::milliseconds tick_interval_ms,
                       const bool debug_print_to_stderr = false)
```

Comment:
This would be a great idea outside Current. With Current, the benefit of having parameters a `CURRENT_STRUCT` and being able to output it via some HTTP endpoint has clear advantages, and `CURRENT_STRUCT`-s, unfortunately, can not be declared inside other `struct`-s.